guides/common/modules/con_prerequisites-for-configuring-project-with-wildfly-keycloak-authentication.adoc

```diff
@@ -3,18 +3,18 @@
 
 Before configuring {Project} with {Keycloak} external authentication, ensure that you meet the following requirements:
 
-* A working installation of {Keycloak} server that uses HTTPS instead of HTTP.
-* A {Keycloak} account with admin privileges.
-* A realm for {Project} user accounts created in {Keycloak}.
-* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store.
-* Users imported or added to {Keycloak}.
+* A working installation of {Keycloak} server that uses HTTPS instead of HTTP
+* A {Keycloak} account with admin privileges
+* A realm for {Project} user accounts created in {Keycloak}
+* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store
+* Users imported or added to {Keycloak}
```

Comment:
```suggestion
* A working installation of {Keycloak} server that uses HTTPS instead of HTTP.
* A {Keycloak} account with admin privileges.
* A realm for {Project} user accounts created in {Keycloak}.
* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store.
* Users imported or added to {Keycloak}.
```